cluster-autoscaler/config/static/autoscaling_options.go

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
'static' part of the path feels sort of redundant. There isn't that much stuff in config/, not sure if it's worth creating package for this single file.

'config/dynamic' exists for historical reasons only, all it contains this days is a function for parsing a string, nothing I would call "dynamic config"

Reply:
Done